model/actors/power/chainpower.go

```diff
@@ -9,22 +9,26 @@ import (
 )
 
 type ChainPower struct {
```

Comment:
The fields in the power actor state have changed, this change accommodates that.